src/common/column/src/types/native.rs

```diff
@@ -268,6 +267,33 @@ impl NativeType for days_ms {
 #[repr(C)]
 pub struct months_days_micros(pub i128);
 
+const MICROS_PER_DAY: i64 = 24 * 3600 * 1_000_000;
+const MICROS_PER_MONTH: i64 = 30 * MICROS_PER_DAY;
```

Comment:
The number of days in each month is different, can we use a fixed number?

Reply:
> The number of days in each month is different, can we use a fixed number?

Well, there's no need to think about it. The same is true in other implementations of the interval type of db. Like duckdb and snowflake